app/client/src/api/services/ConsolidatedPageLoadApi/url.ts

```
@@ -0,0 +1,25 @@
+
+    return `${this.VIEW_URL}?${queryParams}`;
+  };
+
+  static getEditUrl = (requestParams: ConsolidatedApiParams) => {
+    // Remove undefined values from the requestParams object
+    const queryParamsObject = pickBy(requestParams, identity);
+    const queryParams = new URLSearchParams(queryParamsObject);
+
+    return `${this.EDIT_URL}?${queryParams}`;
+  };
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)